dockerfiles/Dockerfile.dev

```diff
@@ -0,0 +1,34 @@
+FROM quay.io/app-sre/qontract-reconcile-builder:0.2.0 as build-image
+
+WORKDIR /work
+
+COPY e2e_tests e2e_tests
+COPY reconcile reconcile
+COPY tools tools
+COPY setup.py .
+COPY dockerfiles/hack/run-integration.py .
+
+RUN python3 -m pip install --no-cache-dir --upgrade pip setuptools wheel && \
+    python3 -m pip wheel . --wheel-dir /work/wheels
+
+
+FROM quay.io/app-sre/qontract-reconcile-base:0.7.1
+
+# Cache mount. We don't need te wheel files in the final image.
+# This COPY will create a layer with all the wheel files to install the app.
+# This layer is not needed in the final image, so we can leverage a cache mount
+# to get rid of it. Implement the cache mount un the RUN command when we are able to
+# to use the buildkit features.
+COPY --from=build-image /work/ /tmp/work/
+#RUN --mount=type=cache,target=/tmp/work/,from=build-image,source=/work \
+
+COPY . /work
+WORKDIR /work
+
+RUN microdnf upgrade -y && \
```

Comment:
If this is for local development, please don't upgrade on every run. It wastes about one minute of execution time, and developers are impatient.

We have an ongoing discussion on how to keep our images reasonably up-to-date without requiring to upgrade on every development build.

Reply:
Good point. Removed it.